`make data` failing

Seems to be caused by the changes to env

```
./env/bin/honcho -e defaults.env,local.env run ./env/bin/fake_data fake_data                          
Traceback (most recent call last):                                                                    
  File "./env/bin/fake_data", line 9, in <module>                                                     
    load_entry_point('gittip==12.3.2-dev', 'console_scripts', 'fake_data')()                          
  File "/home/rummik/source/gittip/www.gittip.com/gittip/utils/fake_data.py", line 214, in main       
    populate_db(wireup.db())                                                                          
TypeError: db() takes exactly 1 argument (0 given)                                                    
make: *** [data] Error 1                                                                              
```

--- a/bin/deactivate-final-rename.py
+++ b/bin/deactivate-final-rename.py
@@ -26,7 +26,7 @@ if len(sys.argv) < 3:
 else:
     first_eight = sys.argv[2]
 
-db = wireup.db()
+db = wireup.db(wireup.env())
 
 target = Participant.from_username(username)
 

--- a/bin/final-gift.py
+++ b/bin/final-gift.py
@@ -14,7 +14,7 @@ from decimal import ROUND_DOWN, Decimal as D
 from gittip import wireup
 from gittip.models.participant import Participant
 
-db = wireup.db()
+db = wireup.db(wireup.env())
 
 username = sys.argv[1] # will fail with KeyError if missing
 tipper = Participant.from_username(username)

--- a/bin/masspay.py
+++ b/bin/masspay.py
@@ -102,7 +102,7 @@ class Payee(object):
 
 def compute_input_csv():
     from gittip import wireup
-    db = wireup.db()
+    db = wireup.db(wireup.env())
     participants = db.all("""
 
         SELECT participants.*::participants

--- a/bin/set-paypal-email.py
+++ b/bin/set-paypal-email.py
@@ -24,7 +24,7 @@ if len(sys.argv) < 4:
 else:
     first_eight = sys.argv[3]
 
-db = wireup.db()
+db = wireup.db(wireup.env())
 
 FIELDS = """
         SELECT username, api_key, paypal_email

--- a/bin/untip.py
+++ b/bin/untip.py
@@ -15,7 +15,7 @@ from gittip import wireup
 
 tippee = sys.argv[1] # will fail with KeyError if missing
 
-db = wireup.db()
+db = wireup.db(wireup.env())
 
 tips = db.all("""
 

--- a/bin/update-user-info.py
+++ b/bin/update-user-info.py
@@ -13,7 +13,7 @@ import requests
 from gittip import wireup
 from requests_oauthlib import OAuth1
 
-db = wireup.db()
+db = wireup.db(wireup.env())
 
 def update_twitter():
     oauth = OAuth1( os.environ['TWITTER_CONSUMER_KEY']

--- a/gittip/utils/fake_data.py
+++ b/gittip/utils/fake_data.py
@@ -211,7 +211,7 @@ def populate_db(db, num_participants=100, num_tips=200, num_teams=5, num_transfe
 
 
 def main():
-    populate_db(wireup.db())
+    populate_db(wireup.db(wireup.env()))
 
 if __name__ == '__main__':
     main()

